### ISSUE TITLE
Return submitHandler result as handle() result

### ISSUE THREAD

**Issue by jzaefferer** [jzaefferer/jquery-validation#757]
Via email:

> I'd like to suggest a change to the handle function that invokes submitHandler please.

> When submitHandler exists, handle currently returns false. Could it instead return the result of submitHandler, so that propagation of the submit event need not be blocked by the presence of submitHandler?

> My use case is that calling form.submit() in submitHandler doesn't work for me, as this causes a normal postback whereas I want to trigger an unobtrusive Ajax postback (via jquery.unobtrusive-ajax.js). The change means I can return true from submitHandler and it works.

> Without this change to handle(), my workaround would be for submitHandler to set itself to null then triggering the submit event using setTimeout (the delay allows handle() to finish and gets round the recursion).

**Comment by jzaefferer** [comment]
I'm sorry for the lack of activity on this issue. Instead of leaving it open any longer, I decided to close old issues without trying to address them, to longer give the false impression that it will get addressed eventually.

To the reporter (or anyone else interested in this issue): If you're affected by the same issue, consider opening a new issue, with a testpage that shows the issue. Even better, try to fix the issue yourself, and improve the project by sending a pull request. This may seem daunting at first, but you'll likely learn some useful skills that you can apply elsewhere as well. And you can help keep this project alive. [We've documented how to do these things, too](https://github.com/jzaefferer/jquery-validation/blob/master/CONTRIBUTING.md). A patch is worth a thousand issues!

**Comment by jzaefferer** [comment]
There's a commit referenced above, but without a PR. Probably lacks tests though.

**Comment by jzaefferer** [comment]
Was fixed by 8b2f1e09b88d3fbfa997a04111d5ad6df92c0e59
